pvlib/financial.py

```diff
@@ -0,0 +1,146 @@
+import numpy as np
+
+
+def lcoe(production=None, cap_cost=None, fixed_om=None):
+
+    """
+    Real levelized cost of electricity as described on pp.43 and 47-48 by
+    [1](https://www.nrel.gov/docs/legosti/old/5173.pdf)
+    Parameters
+    ----------
+    production : np.array, or pd.Series, default None
+        Annual production [kWh/kW installed]
+    cap_cost : np.array, or pd.Series, default None
+        Initial and annual payments on capital costs using real discount rates
+        [$/kW installed]
+    fixed_om : np.array, or pd.Series, default None
+        Annual payments on operations and maintenance costs [$/kW installed]
+
+    Returns
+    ----------
+    LCOE [cents/kWh]
+
+    References
+    ----------
+    .. [1] W. Short, D. J. Packey, and T. Holt, "A Manual for the Economic
+    Evaluation of Energy Efficiency and Renewable Energy Technologies",
+    NREL/TP-462-5173, 1995.
+    """
+
+    cost = cap_cost + fixed_om
+    return np.nansum(cost)*100/np.nansum(production)
```

Comment:
To @kanderso-nrel 's point, this formula is different from that given in the ATB, which adds in variable O&M and fuel cost.

I think we should write out the LCOE formula in the docstring.

Reply:
Thanks @cwhanse, adding LCOE formula to docstring